src/test/java/org/opensearch/sdk/sample/helloworld/TestHelloWorldExtension.java

```diff
@@ -245,11 +245,8 @@ public void onFailure(Exception e) {
         assertEquals("failed to find action [" + UnregisteredAction.INSTANCE + "] to execute", ex.getMessage());
     }
 
+    @Test
     public void testValidatedSettings() {
-        final String expected = randomAlphaOfLengthBetween(1, 5);
-        final String actual = VALIDATED_SETTING.get(Settings.builder().put(VALIDATED_SETTING.getKey(), expected).build());
-        assertEquals(expected, actual);
-
         final IllegalArgumentException exception = expectThrows(
             IllegalArgumentException.class,
             () -> VALIDATED_SETTING.get(Settings.builder().put("custom.validated", "it's forbidden").build())
```

Comment:
Is there any difference if I use "custom.validated" or "custom.validate" , just because of "custom.validated" my test doesn't pass

Reply:
> Is there any difference if I use "custom.validated" or "custom.validate" , just because of "custom.validated" my test doesn't pass

The setting name is defined in the sample class that you are testing, so "validated" is not an existing setting.
https://github.com/opensearch-project/opensearch-sdk-java/blob/ebc684ae223e87f1e05712e086fc1dab911a3b5f/src/main/java/org/opensearch/sdk/sample/helloworld/ExampleCustomSettingConfig.java#L25-L30

Reply:
(and you probably need to add the boolean to that sample, and add another sample with the opposite boolean)